addons/mail/static/src/core/common/composer.js

```diff
@@ -164,7 +164,7 @@ export class Composer extends Component {
                     extraClass: "o-mail-Composer-dropzone",
                     onDrop: this.onDropFile,
                 },
-                () => this.allowUpload
+                () => this.props.allowUpload
```

Comment:
This doesn't change anything since we have a getter for `allowUpload` that aliases the props.